### PR TITLE
update header based on migration in create

### PR DESCRIPTION
**Pull request by wraeth-eth**
## What does this PR do and why?

_Provide a description of what this PR does. Link to any relevant GitHub issues, Notion tasks or Discord discussions._

## Screenshots or screen recordings

_If applicable, provide screenshots or screen recordings to demonstrate the changes._

## Acceptance checklist

- [ ] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
- [ ] I have tested this PR in [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).
- [ ] I have tested this PR in dark mode and light mode (if applicable).

### CODE REVIEW

**Comment by wraeth-eth** [comment]
Current dependencies on/for this PR:
* main
  * **PR #2623** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2623" target="_blank">Graphite</a> 
    * **PR #2671** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2671" target="_blank">Graphite</a> 
      * **PR #2672** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2672" target="_blank">Graphite</a> 
        * **PR #2673** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2673" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2673?utm_source=stack-comment).

File: src/components/Create/Create.tsx [hunk 1]
```diff
@@ -46,7 +46,11 @@ export function Create() {
       <TransactionProvider>
         <V2V3CurrencyProvider>
           <h1 className="text-center text-base font-medium uppercase text-black dark:text-slate-100">
-            <Trans>Create a project</Trans>
+            {!isMigration ? (
+              <Trans>Create a project</Trans>
+            ) : (
+              <Trans>Relaunch a project</Trans>
```
Comment (tomquirk) [inline src/components/Create/Create.tsx:52]: Relaunch or re-launch? No strong opinion but lets pick one

Comment (tomquirk) [inline src/components/Create/Create.tsx:52]: also
```suggestion
              <Trans>Relaunch project</Trans>
```

**Review by tomquirk — APPROVED** [review]
Great call, love this!

**Comment by wraeth-eth** [comment]
**Graphite Merge Job**

Current status: ⏳ Queued to merge

This pull request is currently queued to be merged as part of a stack.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2673?utm_source=stack-comment).


Job Reference: bNgIkwBqux2UJLmdY4Uy

**Comment by wraeth-eth** [comment]
**Graphite Merge Job**

Current status: ⏳ Queued to merge

This pull request is currently queued to be merged as part of a stack.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2673?utm_source=stack-comment).


Job Reference: nnbd1Zy8vknQYYi7LCtZ

**Pull request merged**